src/libraries/Microsoft.Extensions.Configuration.Binder/Microsoft.Extensions.Configuration.Binder.sln

```diff
@@ -1,4 +1,8 @@
-﻿Microsoft Visual Studio Solution File, Format Version 12.00
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
My VS also keeps auto-updating the sln files in this repo. I'm conflicted about whether to include the changes in my PRs. @ViktorHofer what's the recommended course of action here?

Reply:
Please revert any sln header updates. We asked the VS team to stop auto-updating the format version but weren't successfully yet.

Reply:
Ok, I'll revert this file.

Reply:
Done the reverting.